vllm/block.py

```diff
@@ -2,6 +2,7 @@
 from typing import List
 
 from vllm.utils import Device
+from time import monotonic
```

Comment:
Good suggestion. I've removed this call because @zhuohan123 suggested switching over to -1 for the initial last accessed time. Looks like all other calls are already time.monotonic() so we should be good there.